ui/app/AppLayouts/Profile/panels/ProfileShowcasePanel.qml

```diff
@@ -321,22 +321,31 @@ DoubleFlickableWithFolding {
 
         visible: d.dragItem && d.dragItem.showcaseMaxVisibility >= showcaseVisibility
 
-        background: ShapeRectangle {
-            path.strokeColor: dropArea.containsDrag ? Theme.palette.primaryColor2 : Theme.palette.directColor7
-            path.fillColor: dropArea.containsDrag ? Theme.palette.primaryColor3 : Theme.palette.getColor(Theme.palette.baseColor4, 0.7)
-
-            DropArea {
-                id: dropArea
+        background: Rectangle {
```

Comment:
Hmm nothing wrong... but I think it would be better to add this feature (can be optional) directly to `ShapeRectangle`, wdyt?

Reply:
Do you think it's a general bug in the `ShapeRectangle`?

Reply:
I think so... at least I assume we'll want this stroke behavior elsewhere too

Reply:
ahh, we can't really make it work nicely because there's cases where we need a different radius for each corner..